recipe/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "recipe",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npm run build:live",
+    "build": "tsc -p .",
```

Comment:
Change this to `tsc -p ./dist` I believe it should hide the compiled files which we shouldn't be committing.